VeryImportantComponent.js

```diff
@@ -0,0 +1,22 @@
+import { Button } from 'antd';
+import { useCallback } from 'react';
+
+function VeryImportantComponent () {
+  const text = 'Select a number from 1 to 10';
+
+  const clickButton = useCallback(() => {
+    // performs a very important function
+  }, []);
+
+  return (
+    <div style={{ display: 'flex', justifyContent: 'center' }}>
+      <Button
+        onClick={() => clickButton()}
```

Comment:
`IN-HOUSE`: Use the callback function directly instead of using an arrow function